stable/azuredisk-csi-driver/values.yaml

```diff
@@ -1,14 +1,46 @@
 image:
-  repository: mcr.microsoft.com/k8s/csi/azuredisk-csi
-  tag: v0.4.0
-  pullPolicy: Always
+  azuredisk:
+    repository: mcr.microsoft.com/k8s/csi/azuredisk-csi
+    tag: v0.5.0
+    pullPolicy: Always
+  csiProvisioner:
```

Comment:
I would say we should keep the quay.io (quay.io/k8scsi) registry here for all non created microsoft images, instead of the microsoft registry just acting as a mirror, where an upload could be gone wrong

Reply:
please, share the reasons. the microsoft registry is what the community is using.

Reply:
`quay.io/k8scsi` is where the `kubernetes-csi` (see https://github.com/kubernetes-csi) components get official released.

Reply:
alright, that makes sense. I though you were talking about `epository: mcr.microsoft.com/k8s/csi/azuredisk-csi` which is what GH was showing me.